apps/wing/src/cli.ts

```diff
@@ -162,6 +162,12 @@ async function main() {
     .hook("preAction", collectAnalyticsHook)
     .action(runSubCommand("docs"));
 
+  program
+    .command("console-app-path")
+    .description("Print the path to the Wing Console App package")
+    .hook("preAction", collectAnalyticsHook)
+    .action(runSubCommand("consoleAppPath"));
+
```

Comment:
I wonder if there's a way to hide the command from the `help` command 🤔

Reply:
I commented the command since we don't use it for now.